packages/tocco-ui-showcase/src/components/ShowCaseApp.spec.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import {mount} from 'enzyme'
+import ShowCaseApp from './ShowCaseApp'
+
+const cpn = () => ''
+cpn.propTypes = {}
+
+var cpnRaw = `
+import React from 'react'
+import classNames from 'classnames'
+
+/**
+ * SampleComponent
+ */
+const SampleComponent = props => <div/>
+
+SaveButton.propTypes = {
+  /**
+   * some text
+   */
+  sample: React.PropTypes.string.isRequired,
```

Comment:
remove trailing comma to comply with our linting rules (which are not checked in this string)